src/fides/api/models/messaging_template.py

```diff
@@ -56,13 +60,71 @@
 }
 
 
+def _link_properties_to_messaging_template(
+    db: Session,
+    properties: List[Dict[str, Any]],
+    messaging_template: MessagingTemplate,
+) -> Optional[List[Property]]:
+    """
+    Link supplied properties to MessagingTemplate and unlink any properties not supplied.
+    """
+    if len(properties) == 0:
+        return []
```

Comment:
Is your intent to unlink properties not supplied, per the docstring?

There's a case If a MessagingTemplate has some number of existing properties and you update them to have no properties, it will exit here, without removing the properties

Reply:
nice changes here -